config/samples/core_v1beta1_openstackcontrolplane_collapsed_cell.yaml

```diff
@@ -11,7 +11,8 @@ spec:
       secret: osp-secret
   mariadb:
     template:
```

Comment:
ugh sorry this should be `templates`. Fixing it in https://github.com/openstack-k8s-operators/openstack-operator/pull/391 ...